src/unitxt/metrics.py

```diff
@@ -989,9 +1006,7 @@ def compute(
         kendall_results = self.kendalltau(references, predictions, variant=self.variant)
         corr = kendall_results.correlation
         return {
-            "score_name": self.main_score,
```

Comment:
just checking - why are these removed?

Reply:
because they are set outside anyway if globalmetric:
```
def _compute(
        self,
        references: List[List[str]],
        predictions: List[str],
        additional_inputs: List[Any],
    ) -> dict:
        result = self.compute(references, predictions, additional_inputs)
        result["score"] = result[self.main_score]
        result["score_name"] = self.main_score
        return result
```